grpcwebclient/runner.ts

```diff
@@ -0,0 +1,150 @@
+// Copyright 2023-2024 The Connect Authors
```

Comment:
@timostamm I know you mentioned you didn't like the name `runner.ts`. I'm open to call this whatever.

Reply:
You have a `bin/grpcwebclient`, which simply delegates to this file, so I'd just call it `grpcwebclient.ts`.